db/seeds.rb

```diff
@@ -0,0 +1,3 @@
+News.create(title: 'New title 1', description: Faker::Lorem.paragraph(4))
+
+News.create(title: 'New title 2', description: Faker::Lorem.paragraph(5))
```

Comment:
Why do you need these seeds?

Reply:
To add ability for others populate db faster

Reply:
Use loop if you need to create more than one instance